 1. git version 2.34.1
2. user.name=Cayden Anthony
user.email=ca499921@ohio.edu
core.repositoryformatversion=0
core.filemode=true
core.bare=false
core.logallrefupdates=true
remote.origin.url=https://github.com/ca499921/git-lab.git
remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
branch.main.remote=origin
branch.main.merge=refs/heads/main
user.email=ca499921@gmail.com
3. It provides a list of details and features for the command
4. No commits yet

Changes to be committed:
  (use "git rm --cached <file>..." to unstage)

	new file:   README.md

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	answers.md

5. No commits yet

Changes to be committed:
  (use "git rm --cached <file>..." to unstage)

	new file:   README.md
	new file:   answers.md
6. I did the steps out of order so 5 is my answer for #6
7. nothing to commit working tree clean

8. commit 6ae10ed4c799ea9e822eb455a9ee19e193fe483d (HEAD -> master)
Author: Cayden Anthony <ca499921@gmail.com>
Date:   Tue Jan 24 19:08:08 2023 -0500

    Initial commit
9. On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean

10. no

11. error: failed to push some refs to 'https://github.com/ca499921/git-lab.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

12. yes
cs2400
section 107 lab tues
commit

13..  ..  git-lab-2  Labs














